examples/Makefile

```diff
@@ -32,6 +31,43 @@ else
     LDD = ldd
 endif
 
+#-------------------------------------------------------------------------------
+# BLAS and LAPACK
+
+ifeq ($(blas),mkl)
+    # MKL on MacOS doesn't include ScaLAPACK; use default.
+    # For others, link with appropriate version of ScaLAPACK and BLACS.
+    ifneq ($(macos),1)
+        ifeq ($(mkl_blacs),openmpi)
+            ifeq ($(blas_int),int64)
+                scalapack_libs ?= -lmkl_scalapack_ilp64 -lmkl_blacs_openmpi_ilp64
+            else
+                scalapack_libs ?= -lmkl_scalapack_lp64 -lmkl_blacs_openmpi_lp64
+            endif
+        else
+            ifeq ($(blas_int),int64)
+                scalapack_libs ?= -lmkl_scalapack_ilp64 -lmkl_blacs_intelmpi_ilp64
+            else
+                scalapack_libs ?= -lmkl_scalapack_lp64 -lmkl_blacs_intelmpi_lp64
+            endif
+        endif
+    endif
+else ifeq ($(blas),essl)
+    # IBM ESSL
+    # todo threaded, int64
+    # hmm... likely LAPACK won't be int64 even if ESSL is.
+    LIBS += -lessl -llapack
+else ifeq ($(blas),openblas)
+    # OpenBLAS
+    LIBS += -lopenblas
```

Comment:
Usually, there's no need here to link with the actual BLAS library (ESSL, OpenBLAS, etc.). It should be handled by the shared libraries. That is, libslate, libblaspp, liblapackpp are already linked with ESSL, OpenBLAS, etc. and will automatically pull in those dependencies.